src/cluster.c

```diff
@@ -4235,6 +4293,8 @@ sds clusterGenNodesDescription(int filter) {
         ci = sdscatlen(ci,"\n",1);
```

Comment:
Can we just free pre-computed value here? I don't think we need to do another loop.

Reply:
Thanks, let me do it